<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>725</width>
    <height>522</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralWidget">
   <layout class="QVBoxLayout" name="verticalLayout">
    <item>
     <widget class="QLabel" name="label">
      <property name="text">
       <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p align=&quot;center&quot;&gt;&lt;span style=&quot; font-size:14pt; font-weight:600; text-decoration: underline;&quot;&gt;单词检索统计程序&lt;/span&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QTextEdit" name="textEdit"/>
    </item>
    <item>
     <widget class="QTextBrowser" name="textBrowser"/>
    </item>
   </layout>
  </widget>
  <widget class="QMenuBar" name="menuBar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>725</width>
     <height>21</height>
    </rect>
   </property>
   <widget class="QMenu" name="menu_F">
    <property name="title">
     <string>新建文件</string>
    </property>
    <addaction name="newAction"/>
    <addaction name="saveAction"/>
   </widget>
   <widget class="QMenu" name="menu_E">
    <property name="title">
     <string>给定单词计数</string>
    </property>
    <addaction name="Search_1"/>
   </widget>
   <widget class="QMenu" name="menu_B">
    <property name="title">
     <string>检索给定单词</string>
    </property>
   </widget>
   <widget class="QMenu" name="menu_O">
    <property name="title">
     <string>打开</string>
    </property>
    <addaction name="openAction_3"/>
   </widget>
   <addaction name="menu_F"/>
   <addaction name="menu_O"/>
   <addaction name="menu_E"/>
   <addaction name="menu_B"/>
  </widget>
  <widget class="QToolBar" name="mainToolBar">
   <attribute name="toolBarArea">
    <enum>TopToolBarArea</enum>
   </attribute>
   <attribute name="toolBarBreak">
    <bool>false</bool>
   </attribute>
  </widget>
  <widget class="QStatusBar" name="statusBar"/>
  <action name="newAction">
   <property name="text">
    <string>新建(&amp;N)</string>
   </property>
  </action>
  <action name="openAction">
   <property name="text">
    <string>打开(&amp;O)</string>
   </property>
  </action>
  <action name="saveAction">
   <property name="text">
    <string>另存为(&amp;S)</string>
   </property>
  </action>
  <action name="openAction_2">
   <property name="text">
    <string>打开(&amp;O)</string>
   </property>
  </action>
  <action name="Search_1">
   <property name="text">
    <string>检索(&amp;F)</string>
   </property>
  </action>
  <action name="openAction_3">
   <property name="text">
    <string>Open(&amp;O)</string>
   </property>
  </action>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources/>
 <connections/>
</ui>
